modules/contact/test/tests/bouncing-block-contact/frictionless-nodal-reduced-active-set.i

```diff
@@ -0,0 +1,124 @@
+starting_point = .5
+
+[GlobalParams]
+  displacements = 'disp_x disp_y'
+  diffusivity = 1e0
+[]
+
+[Mesh]
+  file = square-blocks-no-offset.e
+[]
+
+[Variables]
+  [./disp_x]
+  [../]
+  [./disp_y]
+  [../]
+[]
+
+[ICs]
+  [./disp_y]
+    block = 2
+    variable = disp_y
+    value = ${starting_point}
+    type = ConstantIC
+  [../]
+[]
+
+[Kernels]
+  [./disp_x]
+    type = MatDiffusion
+    variable = disp_x
+  [../]
+  [./disp_y]
+    type = MatDiffusion
+    variable = disp_y
+  [../]
+[]
+
+
+[Constraints]
+  [./disp_x]
+    type = RANFSNormalMechanicalContact
+    slave = 10
+    master = 20
+    variable = disp_x
+    master_variable = disp_x
+    component = x
+  [../]
+  [./disp_y]
+    type = RANFSNormalMechanicalContact
+    slave = 10
+    master = 20
+    variable = disp_y
+    master_variable = disp_y
+    component = y
+  [../]
+[]
+
+[BCs]
+  [./botx]
+    type = DirichletBC
+    variable = disp_x
+    boundary = 40
+    value = 0.0
+  [../]
+  [./topx]
+    type = DirichletBC
+    variable = disp_x
+    boundary = 30
+    value = 0.0
+  [../]
+  [./boty]
+    type = DirichletBC
+    variable = disp_y
+    boundary = 40
+    value = 0.0
+  [../]
+  [./topy]
+    type = FunctionDirichletBC
+    variable = disp_y
+    boundary = 30
+    function = '${starting_point} - t'
+  [../]
+[]
+
+[Executioner]
+  type = Transient
+  num_steps = 1
+  dt = 1
+  dtmin = 1
+  solve_type = 'PJFNK'
+  petsc_options = '-snes_converged_reason -ksp_converged_reason -ksp_monitor_true_residual -pc_svd_monitor -snes_view'
+  petsc_options_iname = '-mat_mffd_err -pc_type'
+  petsc_options_value = '1e-5          svd'
```

Comment:
why use SVD? It is really hard to solve?

Reply:
This is just to verify that the setup is non-singular

Reply:
I could perhaps change the default `-pc_type` here and then just specify `cli_args` from the `tests` spec

Reply:
K I changed the input file to use amg